schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+            "type": "object",
+            "properties": {
+              "author_comments": {
+                "Title": "Author comments",
+                "description": "Comments from authors.",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "administrator_comments": {
+                "title": "Administrator comments",
+                "description": "Comments from by arXiv administrators",
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/comment"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              },
+              "legacy_comments": {
+                "description": "Unstructured comments from authors and/or arXiv administrators.",
+                "$ref": "#/definitions/comment"
+              }
+            }
+          },
+          "journal_ref": {
+            "title": "Journal reference",
+            "type": "string",
+            "minLength": 1
+          },
+          "publisher_dois": {
+            "title": "Publisher DOIs",
+            "description": "The journal DOI(s).",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/doi"
+            },
+            "minItems": 1
+          },
+          "arxiv_doi": {
+            "title": "arXiv DOI",
+            "description": "The arXiv-issued DOI.",
+            "$ref": "#/definitions/doi"
+          },
+          "report_num": {
+            "title": "Report number",
+            "type": "string",
+            "minLength": 1
+          },
+          "msc_classes": {
+            "title": "List of MSC classes",
+            "description": "Classifications from American Mathematical Society Mathematical Subject Classification (MSC).",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "Primary 14N35, Secondary 14H10, 14H20, 14J26",
+              "Primary 46F30. Secondary: 44A10, 46S10, 46F10, 12J15, 12J25, 16W60",
+              "46B20, 47H10, 54B20, 54F15, 68U05"
+            ],
+            "$comment": "TODO: consider array of an enumerated definition instead. Legacy input is inconsistent."
+
+          },
+          "acm_classes": {
+            "title": "List of ACM classes",
+            "description": "Classifications from ACM Computing Classification System.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "C.2.0; C.2.2; C.2.3; C.2.6",
+              "I.3.5",
+              "G.1.6; F.1.1; I.2.6; G.1.3"
+            ],
+            "$comment": "TODO: consider array of an enumerated definition instead. Legacy input is inconsistent."
+          },
+          "author_links": {
```

Comment:
It would be good to rename this. It make me think this is a link to information about the author.